src/main/java/Main.java

```diff
@@ -0,0 +1,8 @@
+import bot.Sailfish;
+
+public class Main {
+    public static void main(String[] args) {
+        Sailfish fish = new Sailfish();
+        fish.takeControl();
```

Comment:
It is good that there is Single Level of Abstraction in the main which made the main code short and neat.